config/config-network.yaml

```diff
@@ -98,7 +98,8 @@ data:
     # entirely from the template. When choosing a new value be thoughtful
     # of the potential for conflicts - for example, when users choose to use
     # characters such as `-` in their service, or namespace, names.
-    #
+    # {{.Annotations}} can be used for any customization in the go template if needed.
+    # We strongly recommend keeping namespace part of the template to avoid domain name clashes
```

Comment:
Can we get an example of how to use this here?

Reply:
Added an example